aws-predictions/src/androidTest/java/com/amplifyframework/predictions/aws/AWSPredictionsInterpretTest.java

```diff
@@ -44,16 +51,28 @@
  */
 public final class AWSPredictionsInterpretTest {
 
-    private static SynchronousPredictions predictions;
+    private SynchronousPredictions predictions;
 
     /**
-     * Configure Amplify and Predictions plugin before the tests.
-     * @throws Exception if configuration fails
+     * Initialize mobile client singleton.
+     * @throws Exception if mobile client initialization fails
      */
     @BeforeClass
     public static void setUpOnce() throws Exception {
-        TestConfiguration.configureIfNotConfigured();
-        predictions = SynchronousPredictions.delegatingToAmplify();
+        SynchronousMobileClient.instance().initialize();
+    }
+
+    /**
+     * Configure Predictions category before each test.
+     */
+    @Before
+    public void setUp() {
```

Comment:
It should be okay to re-init the mobile client in `@Before`, right? The benefit is, it gets all this setup stuff back into just few lines inside of one method body. Also, a guaranteed baseline before each `@Test` (for the win!)

Singletons and static fields are sort of anti-pattern; I think we should aim to reduce our use of `@BeforeClass` (we use it more than most (quality) code bases I have seen, right now.)

Reply:
Yea, they would do the same thing, but I think leaving it as is is good to remind us that `AWSMobileClient` is a singleton.